my_global_costmap:

    global_frame: map
    robot_base_frame: base_link

    plugins:
        - {name: static_layer, type: "costmap_2d::StaticLayer"}
    static_layer:
        map_topic: map

    footprint: [[-0.05, -0.1], [-0.05, 0.1], [0.05, 0.1], [0.1, 0.0], [0.05, -0.1]]

